### DISCUSSION TITLE
Daha çok reklam için daha çok görev

### DISCUSSION THREAD

**Discussion by erayerdogantr** [IBAX-io/go-ibax#611] [Ideas]
Daha çok reklam yapmak için daha çok satış için daha çok ibax token daha çok görev:)

**Comment by kenpachi-web** [comment]
nası olucak

**Comment by newzat** [comment]
ne zaman gelir

**Comment by Eymenedip** [comment]
Nasıl olacak

**Comment by Eymenedip** [comment]
Acaba ödülü kimler alacak

**Comment by mrtgezgin** [comment]
iki gündür tweet gelmiyor
